### DISCUSSION TITLE
how to write React component demo?

### DISCUSSION THREAD

**Discussion by stekovinbranturry** [facebook/docusaurus#9592] [Q&A]
I cannot simply write demo like this:
<img width="254" alt="image" src="https://github.com/facebook/docusaurus/assets/33021960/4e05203b-c8c5-4e2e-8c1b-ec714412f24e">

then i will get a code demo:
<img width="839" alt="image" src="https://github.com/facebook/docusaurus/assets/33021960/da1dc9ad-835c-40df-acff-c2510c6f116f">

which plugin can help to implement?

appreciated!

**Comment by slorber** [comment]
Documented here: https://docusaurus.io/docs/markdown-features/code-blocks#interactive-code-editor
